detox/test/e2e/24.uidevice.test.js

```diff
@@ -0,0 +1,14 @@
+  describe(':android: UIDevice', () => {
+    beforeEach(async() => {
+      await device.reloadReactNative();
+    });
+
+    it(`:android: device.getUiDevice() + getDisplayHeight() + getDisplayWidth() + click()`, async () => {
+      await element(by.text('Device')).tap();
+      const uiDevice = device.getUiDevice();
+      const height = await uiDevice.getDisplayHeight();
+      const width = await uiDevice.getDisplayWidth();
+      await uiDevice.click(width / 2, height / 2);
+      await expect(element(by.text('Tap works'))).toBeVisible();
+    });
```

Comment:
Wouldn't it be better to add this describe to `e2e/06.device.test.js`?

Reply:
no, Rotem and I talked about it and decide that we prefer to create a test suite dedicated to UIDevice